BMI Calculator/BMI Calculator/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Dynamic;
+
+namespace BMI_Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            UserInfo();
+            UserInfo();
+        }
+
+        static void CalBMI(double weight, double height, string fName, string lName)
```

Comment:
Unnecessary shortage of naming. Calling it `CalculateBMI()` is fine 😜 
Also. When I'd call a method called `CalculateBMI()`, I'd expect it to only calculate it for me and return the result to me.
Separate those processes of "CalculateX()" and printing something into different methods / places 😄